examples/locales/en.ftl

```diff
@@ -0,0 +1,12 @@
+greeting = Hello { $first_name }!
+
+cart = { $first_name }, there {
+  $apples ->
+    [0] are no apples
+    [one] is one apple
+    *[other] are { $apples } apples
+  } in your cart.
+
+checkout = Thank you for purchasing!
+
+language-set = Language has been set to English!
```

Comment:
You can add `de.ftl` with the following content in order to be consistent with the German example in the docs.

```flt
greeting = Hallo { $first_name }!

cart = { $first_name }, es {
  $apples ->
    [0] ist kein Apfel
    [one] ist ein Apfel
    *[other] sind { $apples } Äpfel
  } in deinem Einkaufswagen.

checkout = Danke für deinen Einkauf!

language-set = Die Sprache wurde zu Deutsch geändert!
```
Feel free to keep the Russian version, too, if you find someone who can translate it.

Reply:
Thank you, I'll add that. And we've some more languages on the way :D

Reply:
I think we should encourage everyone add their language to the example so we can see more and cover more usage case.